### PR TITLE
Create test_post_request function

### PR DESCRIPTION
**Pull request by boushrabettir**
- Imported the `json` module to facilitate validation of data in JSON format.
- Created function to assess whether the provided data conforms to JSON format.
- Within the following three functions, implemented a simple conditional using the function created to evaluate incoming POST requests, ensuring their content adheres to valid JSON standards.

The issue can be viewed here #13

### CODE REVIEW

**Review by Pradyuman7 — COMMENTED** [review]
Can you also link the issue in this PR please?

**Comment by boushrabettir** [comment]
> Can you also link the issue in this PR please?

Yes the issue is #13.

**Comment by boushrabettir** [comment]
> Is the build failure expected? And, can you also fix the conflicts in this PR

For every PR I've seen, the build failure is present. Could it be possible to create a new issue for it?

**Review by Pradyuman7 — CHANGES_REQUESTED** [review]
Doesn't the issue mean that you have to implement this functionality for the server and not just for tests? I think the tests are meant to be written as part of the PR to show that the added functionality works correctly

**Comment by wrussell1999** [comment]
Would love to see this resolved before Week 3 @boushrabettir

**Comment by boushrabettir** [comment]
> Doesn't the issue mean that you have to implement this functionality for the server and not just for tests? I think the tests are meant to be written as part of the PR to show that the added functionality works correctly

The issue specifically said before it changed that we implement this into `test_pytest.py`. I'll implement it tonight.

**Comment by boushrabettir** [comment]
> Doesn't the issue mean that you have to implement this functionality for the server and not just for tests? I think the tests are meant to be written as part of the PR to show that the added functionality works correctly

I've fixed it up right now the functionality, and also added in the last `POST request` functionality, it seems no one did it.

File: app.py [hunk 1]
```diff
@@ -1,146 +1,123 @@
...
+            return jsonify({"id": new_education_index})
+
     return jsonify({})
 
 
-@app.route("/resume/skill", methods=["GET", "POST", "DELETE"])
+@app.route('/resume/skill', methods=['GET', 'POST', 'DELETE'])
 def skill():
-    """
+    '''
```
Comment (Pradyuman7) [inline app.py:90]: nit: why are we changing these quotes?

**Review by Pradyuman7 — DISMISSED** [review]

File: app.py (hunk 1 above)
Comment (boushrabettir) [reply app.py:90]: I didn't change them. It was updated from another merged PR. I can go ahead and update them.

Comment (boushrabettir) [reply app.py:90]: Resolved [here](https://github.com/MLH-Fellowship/orientation-project-python-23.FAL/pull/27/commits/7af1ed34505885c1dff9bf0a630b7c68de392c90)

**Review by Pradyuman7 — APPROVED** [review]

**Pull request merged**